examples/lib/language_tour/classes/employee.dart

```diff
@@ -0,0 +1,47 @@
+// ignore_for_file: unnecessary_cast
+// #docplaster
+import '../util/print.dart';
+
+Map getDefaultData() => {}; // stub
+
+// #docregion
+class Person {
+  String firstName;
+
+  Person.fromJson(Map data) {
+    $print('in Person');
+  }
+}
+
+// #docregion method-then-constructor
+class Employee extends Person {
+  // #enddocregion ,
+  Employee() : super.fromJson(getDefaultData());
+  // ···
+  // #enddocregion method-then-constructor
+  // #docregion
+  // Person does not have a default constructor;
+  // you must call super.fromJson(data).
+  Employee.fromJson(Map data) : super.fromJson(data) {
+    $print('in Employee');
+  }
+// #docregion method-then-constructor
+}
+// #enddocregion method-then-constructor
+
+void main() {
+  var emp = new Employee.fromJson({});
+  // Prints:
```

Comment:
This is not some kind of a test, right? It's just a doc. I'm confused why we use `$print` above. Is that tested elsewhere?

Reply:
Oh, never mind, I see that we do actually test this from a `classes_test.dart`. In that case, all good. Maybe I'd remove the comments here since it seems like these are not shown or tested anywhere and will get out of sync easily.

Reply:
> Maybe I'd remove the comments here since it seems like these are not shown or tested anywhere

This code shows up in DartPad IIRC (note that I haven't addressed DartPad code syncs yet; that will be done in a subsequent PR).

Reply:
Okay. Let's keep it then.